cli/crates/cli/tests/graphql/owner/global/todo/todo-delete.graphql

```diff
@@ -0,0 +1,5 @@
+mutation TodoDelete($id: ID!) {
+	todoDelete(by: { id: $id }) {
```

Comment:
Hmm, it'd be good to set up `yarn fmt` for this repo. In the api repo we use two spaces for indentation in .graphql files. 🙂

Reply:
done!